src/healthcheck.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import delay from 'delay'
+import { RTMClient } from '@slack/client'
+import config from 'config'
+import dbApi from './db-api'
+import type { DbApi } from 'icarus-backend'; // eslint-disable-line
+
+const { logger } = config.get('server')
+
+async function fetchBestBlock(db) {
+  return dbApi(db).bestBlock()
+}
+
+async function start(db: any) {
+  console.log('start') // eslint-disable-line
+  const token = process.env.SLACK_TOKEN
+  const channelId = process.env.SLACK_CHANNEL
+  process.env.BLOCK_RESPONSE = 'true'
+  const rtm = new RTMClient(token)
+  rtm.start()
+
+  let responding = true
+  let bestBlock = await fetchBestBlock(db)
+
+  while (true) { // eslint-disable-line
+    await delay(70000) // eslint-disable-line
+    const dbBestBlock = await fetchBestBlock(db) // eslint-disable-line
+    const changed = !(bestBlock === dbBestBlock)
+
+    if (responding !== changed) {
```

Comment:
I would rather put the comparison directly there: `if (bestBlock === dbBestBlock)`